project-base/config/graphql/types/Pagination/ProductConnection.types.yaml

```diff
@@ -6,5 +6,7 @@ ProductConnection:
             totalCount:
                 type: "Int!"
                 description: "Total number of products"
+            productFilterOptions:
+                type: "ProductFilterOptions!"
```

Comment:
Can you please add descriptions to all fields introduced in this PR?

Reply:
f871c4597f880daeac1146c4f45220bbf5a724ae and b9fc5dc35f8b84c68c3e778c4b57aedcb58eb51f